pyroomacoustics/transform/dft.py

```diff
@@ -75,38 +74,48 @@ class DFT(object):
         ``numpy``.
     axis : int, optional
         Axis over which to compute the FFT. Default is first axis.
-    bits : int, optional
-        How many bits to use for real input. Twice the amount will be used for 
-        the complex spectrum. 32 will use ``float32``/``complex64`` and 64 will
-        use ``float64``/``complex128``. Default is 32.
+    precision, bits : string, np.float32, np.float64, np.complex64, np.complex128, optional
+        How many precision bits to use for the input.
+        If 'single'/np.float32/np.complex64, 32 bits for real inputs (64 for complex spectrum).
+        If 'double'/np.float64/np.complex128 (default), 64 bits for real inputs (128 for complex spectrum).
 
     """
 
     def __init__(self, nfft, D=1, analysis_window=None, synthesis_window=None, 
-        transform='numpy', axis=0, bits=32):
+        transform='numpy', axis=0, precision='double', bits=None):
 
         self.nfft = nfft
         self.D = D
         self.axis=axis
-        
-        if bits == 32:
-            time_dtype = 'float32'
-            freq_dtype = 'complex64'
-        elif bits == 64:
-            time_dtype = 'float64'
-            freq_dtype = 'complex128'
+
+        if bits is not None and precision is not None:
+            warnings.warn('Deprecated keyword "bits" ignored in favor of new keyword "precision"', DeprecationWarning)
+        elif bits is not None and precision is None:
+            warnings.warn('Keyword "bits" is deprecated and has been replaced by "precision"')
+            if bits == 32:
+                precision = 'single'
+            elif bits == 64:
+                precision = 'double'
+
+        if precision == np.float32 or precision == np.complex64 or precision == 'single':
```

Comment:
Maybe here as well we should just set to double whenever single is not explicitely required.